datafusion/sqllogictest/test_files/string/string_view.slt

```diff
@@ -91,6 +91,16 @@ select octet_length(column1_utf8view) from test;
 7
 NULL
 
+query IIII
+SELECT
+  BIT_LENGTH(arrow_cast('Andrew', 'Utf8View')),
+  BIT_LENGTH(arrow_cast('datafusion数据融合', 'Utf8View')),
+  BIT_LENGTH(arrow_cast('💖', 'Utf8View')),
+  BIT_LENGTH(arrow_cast('josé', 'Utf8View'))
+;
+----
+48 176 32 40
+
 query error DataFusion error: Arrow error: Compute error: bit_length not supported for Utf8View
```

Comment:
I think we can add a TODO comment here and file an issue to track https://github.com/apache/arrow-rs/pull/6671. After upgrading to the corresponding arrow-rs version, we should address the TODO comment in `string/string_query.slt.part#L1100`.